test/image/mocks/gl3d_text-weirdness.json

```diff
@@ -42,22 +42,6 @@
   ],
   "layout": {
     "autosize": true,
-    "undefined": {
-      "cameraposition": [
-        [
-          0.13855639464070454,
-          0.5365430934822464,
-          0.7678929376547012,
-          -0.32134727420767745
-        ],
-        [
-          0,
-          0,
-          0
-        ],
-        1.8771354322421991
-      ]
-    },
```

Comment:
I thought this might not be necessary. If one wants to enable `cameraposition` it is required to use `"scene"` instead of `"undefined"`! Also now I am curious about this mock & why it was named `weirdness`? Also if we could possibly remove it or reduce the width/height.

Reply:
Wow. Thanks for fixing this. That `undefined` shouldn't be there.